src/controllers/api/v1/subscriptions/index.ts

```diff
@@ -0,0 +1,89 @@
+import { Request, Response, NextFunction } from 'express'
+import Container from 'typedi'
+import SubscriptionService from '../../../../services/subscription'
+import { WALLET_TRANSFER_TO_EVENT } from '../../../../constants/events'
+import UserService from '../../../../services/user'
+import UserSubscriptionService from '../../../../services/userSubscription'
+import RequestError from '../../../../models/RequestError'
+import { validationResult } from 'express-validator'
+
+export default class SubscriptionsController {
+  static async subscribeWalletTransfersTo (req: Request, res: Response, next: NextFunction) {
+    try {
+      const {
+        address,
+        webhookUrl
+      } = req.body
+
+      const errors = validationResult(req)
+      if (!errors.isEmpty()) {
+        res.status(400).json({ error: errors.mapped() })
+        return
+      }
+
+      const userService = Container.get(UserService)
```

Comment:
Can you think of different names for this 3 services? They all names kinda same though functionality differ.
IMO at least one of them need to be changed to make it more clear